ragna/embedding_models/_embedding.py

```diff
@@ -0,0 +1,14 @@
+from ragna.core import Chunk, Embedding, EmbeddingModel
+
+
+class MiniLML6v2(EmbeddingModel):
+    def __init__(self):
+        super().__init__()
+        from chromadb.utils import embedding_functions
+        self.model = embedding_functions.DefaultEmbeddingFunction()
```

Comment:
Let's make this private given that users do not need to interact with it.

```suggestion
        self._model = embedding_functions.DefaultEmbeddingFunction()
```